src/filters/FilterPerfTest.cpp

```diff
@@ -30,14 +30,11 @@ void usage(const char* name);
 template <typename K, typename V>
 constexpr std::unordered_map<V, K> reverseMap(const std::unordered_map<K, V>& map);
 
-enum class FilterType
-{
-	EKF,
-	PoseGraph
+enum class FilterType {
+	EKF
 };
```

Comment:
An example of the weird enum linting contradiction we were talking about, GitHub CI fails

Reply:
I did some research into this, I think the GitHub check just calls the clang-format-11, which defaults to formatting based on just LLVM. Adding the parameter `-style=file` should use the .clang-format in out code.

Reply:
The line:
`diff=$(git clang-format-11 --diff)`
Should be
`diff=$(git clang-format-11 -style=file --diff)`
In the formatted workflow

Reply:
Format check fixed by #295